### PR TITLE
Migrate config inspect test to api test

### PR DESCRIPTION
**Pull request by yongtang**
This fix migrates config inspect test in integration-cli to api test.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

File: integration-cli/docker_cli_config_inspect_test.go [hunk 1]
```diff
@@ -10,29 +10,7 @@ import (
...
-
-	out, err := d.Cmd("config", "inspect", testName)
-	c.Assert(err, checker.IsNil, check.Commentf(out))
-
-	var configs []swarm.Config
-	c.Assert(json.Unmarshal([]byte(out), &configs), checker.IsNil)
-	c.Assert(configs, checker.HasLen, 1)
-}
-
+// TODO: TestConfigInspectMultiple should either be moved to docker/cli or dropped.
```
Comment (dnephin) [inline integration-cli/docker_cli_config_inspect_test.go]: We have pretty good test coverage here: https://github.com/docker/cli/blob/master/cli/command/config/inspect_test.go

I think you can probably remove it now.

**Review by dnephin — APPROVED** [review]
LGTM

File: integration/config/config_test.go [hunk 2]
```diff
@@ -327,3 +328,27 @@ func waitAndAssert(t *testing.T, timeout time.Duration, f func(*testing.T) bool)
 		time.Sleep(100 * time.Millisecond)
 	}
 }
+
+func TestConfigInspect(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
```
Comment (thaJeztah) [inline integration/config/config_test.go]: Why is this skipped on Windows?

**Review by thaJeztah — APPROVED** [review]
LGTM, but one question

File: integration-cli/docker_cli_config_inspect_test.go (hunk 1 above)
Comment (yongtang) [reply integration-cli/docker_cli_config_inspect_test.go]: Thanks @dnephin. Removed.

File: integration/config/config_test.go (hunk 2 above)
Comment (yongtang) [reply integration/config/config_test.go]: @thaJeztah Both config and secret currently works with swarm so it is skipped on on-linux case.

**Review by boaz0 — APPROVED** [review]
LGTM :grinning:

File: integration/config/config_test.go (hunk 2 above)
Comment (thaJeztah) [reply integration/config/config_test.go]: oh, right, had my wires crossed lol

**Comment by arm64b** [comment]
Hi @yongtang , according to my observation, the `powerpc` node is very volatile recently. I've struggled with this node for couple of days, so I just want to comfort you here 🤗

**Comment by yongtang** [comment]
Thanks @arm64b, we out to do something to improve the stability of the build. On the same boat 😅

**Pull request merged**